src/components/results/securityanalysis/security-analysis-result-tab.tsx

```diff
@@ -102,8 +102,25 @@ export const SecurityAnalysisResultTab: FunctionComponent<
         sortWay: SORT_WAYS.asc,
     });
 
+    const resultType = useMemo(() => {
```

Comment:
Why create useMemo for this ?

Reply:
Ah ok it already exist. I think it's counterproductive and useless. Or I missed something ?

Reply:
where does it exist ? I did not see it
I agree we don't have to use "useMemo" since it's only a boolean calculation though, if that was your point

Reply:
You moved the code from line 237, no ?
Yeah nevermind there is useMemo and useCallback everywhere in that file anyway